MapboxVision/src/main/java/com/mapbox/vision/video/videosource/camera/YuvAllocation2Rgb.kt

```diff
@@ -0,0 +1,63 @@
+package com.mapbox.vision.video.videosource.camera
+
+import android.graphics.ImageFormat
+import android.renderscript.Allocation
+import android.renderscript.Element
+import android.renderscript.RenderScript
+import android.renderscript.ScriptIntrinsicYuvToRGB
+import android.renderscript.Type
+import android.util.Size
+import android.view.Surface
+
+internal class YuvAllocation2Rgb(
```

Comment:
What the reason to  rename Yuv2RgbRender to YuvAllocation2Rgb?

Reply:
Well, it doesn't render anything, but instead provides Allocation with Surface, then converts frames to Rgb.
WDYT?